plugins/woocommerce/changelog/update-product_editor_naming

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: tweak
+
+Update copy from 'form' to 'editor' to match documentation.
```

Comment:
Super nitpick. It self-explanatory in another changelog file but here context would be nice:

```suggestion
Product Editor: Update copy from 'form' to 'editor' to match documentation.
```

Reply:
Done as part of: https://github.com/woocommerce/woocommerce/pull/54904/commits/da62dc475743d5832fcde4c15a90cdf5fbc7c896